server/src/main/resources/org/opensearch/bootstrap/security.policy

```diff
@@ -79,6 +79,12 @@ grant codeBase "${codebase.jna}" {
   permission java.lang.RuntimePermission "accessDeclaredMembers";
 };
 
+grant codeBase "${codebase.jackson-databind}" {
```

Comment:
:+1:

Reply:
Unfortunately this is not working for jackson-databind, so we need a workaround or we should look at the new PR #5768.